gallery/how_to/work_with_microtvm/micro_aot.py

```diff
@@ -30,6 +30,29 @@
 or on Zephyr platform on a microcontroller/board supported by Zephyr.
 """
 
+######################################################################
+#
+#     .. include:: ../../../../gallery/how_to/work_with_microtvm/install_zephyr.rst
```

Comment:
Tried to find where else you have included this `install_zephyr.rst`. But didn't find it. Probably you can just move the context of `install_zephyr.rst` to this file?

Reply:
I'd be in favor of this, but I'd love to get thoughts from @mehrdadh.

Reply:
This file should be added to all microTVM Zephyr tutorials. I think some of them are missing in this PR. That's why we refactored it to a separate file